server/db/index.js

```diff
@@ -0,0 +1,30 @@
+var ENV = 'development';
+var DATABASE = 'stockduel';
+
+// ============= Require Schema-builders ============= \\
+var users = require('../schema/authSchema.js');
+var matches = require('../schema/matchesSchema.js');
+var trades = require('../schema/tradesSchema.js');
+var stocks = require('../schema/stocksSchema.js');
+var stock_prices = require('../schema/stockpricesSchema.js');
+var archive = require('../schema/archiveSchema.js');
```

Comment:
Having this be named stock_archive or stock_prices_archive and the schema be named stockarchiveSchema or stockpricesarchiveSchema would make this clearer. It's conceivable (though unlikely, I suppose) that we'll have other forms of archives coming up in the future.